submitit/helpers.py

```diff
@@ -216,3 +217,67 @@ def __enter__(self) -> None:
 
     def __exit__(self, *args):
         os.chdir(self.original_dir)
+
+
+def monitor_jobs(jobs: tp.Sequence[core.Job[core.R]], sleep_time_s: float = 30, test_mode: bool = False):
```

Comment:
@MarcSzafraniec I was thinking of a couple of things yesterday actually:
1. the return type is missing here
2. shouldn't we return jobs directly? currently it looks like it is the index in the list, which is not that convenient to use. So either a tuple of job set/list, or a dict status -> jobs, or actually nothing since it's easy to filter between failed and successful eg: `[j for j in jobs if j.exception() is not None]` or `[j for j in jobs if j.state == "FAILED"]`